docs/octopus-cloud/index.md

```diff
@@ -51,21 +51,52 @@ If you are having difficulty uploading packages to your Octopus built-in package
 
 In the near future, the Octopus Cloud will become available in additional regions and you'll have the ability to move between regions.
 
-## Reset the User Password
+## Reset the Octopus Account Password
+
+If you forget your username or password for your Octopus account or Octopus instance, you can reset them via Octopus.com
+
+1. Visit [Octopus.com/signin](https://Octopus.com/signin).
+1. Click 'Forgot your password?'
+1. Reset your credentials and log into your Octopus account. You can now launch your Octopus instance from the accounts dashboard.
+
+
+## Reset the Instance User Password
+:::hint
+This feature is being deprecated.
```

Comment:
Do you think there is value in calling out directly what feature is deprecating this? For e.g. we could link here https://octopus.com/docs/administration/authentication/octopusid-authentication

Reply:
good point, I've added the link.